Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageCell.swift

```diff
@@ -11,27 +11,36 @@ import UIKit
 public class ChatMessageCell: _TableViewCell, ComponentsProvider {
     public static var reuseId: String { "\(self)" }
 
-    /// The container that holds the date separator and the message content view.
-    /// This is internal since it is a temporary solution.
+    /// The container that holds the header, footer and the message content view.
     internal lazy var containerStackView = UIStackView()
         .withoutAutoresizingMaskConstraints
         .withAccessibilityIdentifier(identifier: "containerStackView")
 
-    /// The date separator view that groups messages from the same day.
-    /// This is internal since it is a temporary solution.
-    internal lazy var dateSeparatorView: ChatMessageListDateSeparatorView = components
-        .messageListDateSeparatorView.init()
+    /// The header View of the cell. It can be used to display additional information
+    /// about the message above the message's content.
+    internal lazy var headerContainerView: UIView = .init()
         .withoutAutoresizingMaskConstraints
-        .withAccessibilityIdentifier(identifier: "dateSeparatorView")
+        .withAccessibilityIdentifier(identifier: "headerContainerView")
 
     /// The message content view the cell is showing.
     public private(set) var messageContentView: ChatMessageContentView?
 
+    /// The footer View of the cell. It can be used to display additional information
+    /// about the message below the message's content.
+    internal lazy var footerContainerView: UIView = .init()
+        .withoutAutoresizingMaskConstraints
+        .withAccessibilityIdentifier(identifier: "footerContainerView")
+
     /// The minimum spacing below the cell.
     public var minimumSpacingBelow: CGFloat = 2 {
         didSet { updateBottomSpacing() }
     }
 
+    /// The spacing between the header, content and footer
+    public var spacingBetween: CGFloat = 8 {
```

Comment:
headerFooterSpacing?

Reply:
If you don't mind i'll update it on the following patch that implements the thread replies counter decoration view